src/middleware/packages/auth/mixins/auth.js

```diff
@@ -14,7 +17,7 @@ const AuthMixin = {
     accountSelection: [],
     accountsDataset: 'settings'
   },
-  dependencies: ['api', 'webid'],
+  dependencies: ['api', 'webid', 'ldp.resource'],
```

Comment:
Even if it's a bit counter-intuitive, It's better to add to dependencies only services which are called on `created` or `started`. Otherwise there is a real risk of circular dependencies (since the service won't start until all dependencies are started)